python-sdk/tests/files/locations/test_ftp.py

```diff
@@ -0,0 +1,54 @@
+import ftplib
+from unittest.mock import Mock, patch
+
+from airflow.providers.ftp.hooks.ftp import FTPHook
+
+from astro.files.locations import create_file_location
+
+
+def test_get_transport_params_for_ftp():  # skipcq: PYL-W0612, PTC-W0065
+    """test get_transport_params() method which should empty dict"""
```

Comment:
docstring doesnt make much sense, can you please correct it?

Reply:
Modified